### PR TITLE
test: Use unittest assert methods

### PR DESCRIPTION
**Pull request by marusak**
These provide better log when they fail.
For example `assert (size > 300000)` provides:
```
    assert (size > 300000)
AssertionError
```

while `assertGreater(size, 300000) provides:
```
    assert (size > 300000)
AssertionError: 1 is not greater than 300000
```

I only kept `assert False, <reason>`.

### CODE REVIEW

File: test/verify/check-metrics [hunk 1]
```diff
@@ -23,7 +23,7 @@ def getMaximumSpike(browser, g_type, saturation, hour, minute):
         sel += ":first-child"
 
     points = browser.attr(sel, "points")
-    assert "NaN" not in points, points
+    self.assertNotIn("NaN", points)
```
Comment (marusak) [inline test/verify/check-metrics:26]: self is not defined

File: test/verify/check-metrics [hunk 2]
```diff
@@ -14,43 +14,43 @@ from testlib import *
 from machine_core.constants import TEST_OS_DEFAULT
 
 
-def getMaximumSpike(browser, g_type, saturation, hour, minute):
+def getMaximumSpike(self, g_type, saturation, hour, minute):
```
Comment (martinpitt) [inline test/verify/check-metrics:17]: This is weird for a non-method. Better call this `test` or so.

**Review by martinpitt — DISMISSED** [review]
Thanks! Especially looking forward to getting better debugging for storage-resize.

File: test/verify/check-metrics (hunk 2 above)
Comment (marusak) [reply test/verify/check-metrics]: You meant like this? (pushed)

Comment (martinpitt) [reply test/verify/check-metrics]: Right, thanks!

**Review by martinpitt — APPROVED** [review]

**Pull request merged**